testng-core/src/main/java/org/testng/SuiteRunner.java

```diff
@@ -37,15 +37,15 @@
       Collections.synchronizedMap(Maps.newLinkedHashMap());
   private final List<TestRunner> testRunners = Lists.newArrayList();
   private final Map<Class<? extends ISuiteListener>, ISuiteListener> listeners =
-      Maps.newConcurrentMap();
+      Maps.newLinkedHashMap();
```

Comment:
Why not synchronizedLinkedHashMap?

Reply:
I dont see a concurrent usage in this case. That is why I didnt use the synchronized variant.

Reply:
If it doesn't break #2063, then ok